docs/ai/conceptual/embeddings.md

```diff
@@ -0,0 +1,64 @@
+
+## Embeddings preserve semantic meaning
+
+Embeddings turn non-numeric data into numeric data that preserves semantic meaning. Specifically, an embedding is an array of numeric values stored as a vector in a database that an LLM uses to compare embeddings. Two embeddings with similar vectors also have similar semantic values, such as an image and its alt text, or a course syllabus and the textbook for that course.
+
+### Choosing embedding models to generate embeddings
+
+You generate embeddings for your raw data by using an embedding model: an AI model that can encode a piece of non-numeric data into a vector (a long array of numbers). The model can also decode an embedding into non-numeric data that has the same or similar meaning as the original, raw data. The embedding model you choose affects which data types you can generate embeddings for.
+
+In Semantic Kernel, you build connectors for embedding models and databases into your kernel, so you can quickly pull in text data, and generate and store embeddings. This lets you use a vector database solution to store and retrieve semantic memories.
+
+In Azure OpenAI, the service includes access to [embedding models](/azure/ai-services/openai/concepts/models#embeddings).
+
+### Store and process embeddings in a vector database
+
+Vector databases are designed to store and process vectors, so they're a natural home for embeddings. Different vector databases offer different processing capabilities, so you should choose one based on your raw data and your goals. For information about your options, see [available vector database solutions](vector-dbs.md#available-vector-database-solutions).
+
+## Use cases for embeddings
+
+This section lists the main use cases for embeddings.
+
+### Increase the amount of text a model will process
+
+Use embeddings to increase the amount of text a model will process, such as during [prompt engineering](prompt-engineering-in-dot-net.md).
+
+For example, suppose you want to include 500 pages of text in a prompt. The number of tokens for that much raw text will exceed the input token limit, making it impossible to directly include in a prompt. You can use embeddings to summarize and break down large amounts of that text into pieces that are small enough to fit in one input, and then assess the similarity of each piece to the entire raw text. Then you can choose a piece that best preserves the semantic meaning of the raw text and use it in your prompt without hitting the token limit.
```

Comment:
You could use a planner to automate it in SK. Plus the Redis code you just wrote seems like it could be used to do this. Also, it kind of sounds like retrieval-augmented generation - I'm waiting for alex to draft the RAG how-to before I finish the RAG conceptual doc, so I'll know more soon.